openml/tasks/task.py

```diff
@@ -7,38 +7,58 @@
 import openml._api_calls
 from ..utils import _create_cache_directory_for_id
 
-
 class OpenMLTask(object):
-    def __init__(self, task_id, task_type_id, task_type, data_set_id,
-                 target_name, estimation_procedure_type, data_splits_url,
-                 estimation_parameters, evaluation_measure, cost_matrix,
-                 class_labels=None):
+    def __init__(self, task_id, task_type_id, task_type, data_set_id, estimation_procedure_type,
+                 estimation_parameters, evaluation_measure):
         self.task_id = int(task_id)
         self.task_type_id = int(task_type_id)
         self.task_type = task_type
         self.dataset_id = int(data_set_id)
-        self.target_name = target_name
         self.estimation_procedure = dict()
         self.estimation_procedure["type"] = estimation_procedure_type
-        self.estimation_procedure["data_splits_url"] = data_splits_url
         self.estimation_procedure["parameters"] = estimation_parameters
         #
         self.estimation_parameters = estimation_parameters
```

Comment:
I would rather store the estimation procedure id (not available in api right now, let's add it)

Reply:
@janvanrijn I'm afraid that won't be backwards compatible... But we can probably transit at some point to store something different in the cache. Do you want to open a new issue on the OpenML issue tracker?